content/reference/reader.adoc

```diff
@@ -209,6 +209,35 @@ by invoking the Var `#'my.project.foo/bar` on the vector `[1 2 3]`. The data rea
 
 Reader tags without namespace qualifiers are reserved for Clojure. Default reader tags are defined in https://clojure.github.io/clojure/clojure.core-api.html#clojure.core/default-data-readers[default-data-readers] but may be overridden in `data_readers.clj` or by rebinding https://clojure.github.io/clojure/clojure.core-api.html#clojure.core/%2Adata-readers%2A[pass:[*data-readers*]]. If no data reader is found for a tag, the function bound in https://clojure.github.io/clojure/clojure.core-api.html#clojure.core/%2Adefault-data-reader-fn%2A[pass:[*default-data-reader-fn*]] will be invoked with the tag and value to produce a value. If pass:[*default-data-reader-fn*] is nil (the default), a RuntimeException will be thrown.
 
+Clojure 1.4 introduced some tagged literals useful for Java interop. So far, these include instants and UUIDs.
+
+Instants have the format `#inst "yyyy-mm-ddThh:mm:ss.fff+hh:mm"`. The default reader will parse the supplied string into a
+`java.util.Date` by default. For example:
+
+----
+(def instant #inst "2018-03-28T10:48:00.000")
+(= java.util.Date (class instant))
+;=> true
+----
+
+Since *data-readers* is a dynamic var that can be bound, you can replace the default reader with a different one. For example, `clojure.instant/read-instant-calendar` will parse the literal into a `java.util.Calendar`, while `clojure.instant/read-instant-timestamp` will parse it into a `java.util.Timestamp`:
```

Comment:
Need markup around `*data-readers*` or it will be treated as asciidoc emphasis. Can use `pass:[*data-readers*]` to tell asciidoc to pass the string as is.